manifests/cluster-network-addons/0.4.0/operator.yaml

```diff
@@ -131,6 +131,8 @@ spec:
           value: docker.io/nfvpe/multus:latest
         - name: LINUX_BRIDGE_IMAGE
           value: quay.io/kubevirt/cni-default-plugins:latest
+        - name: BRIDGE_MARKER_IMAGE
+          value: quay.io/kubevirt/bridge-marker          
```

Comment:
please release bridge marker with a tag and keep tagged version here

Reply:
added 0.1.0